src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -113,8 +113,19 @@ private static HashSet<string> CreateTargetsSet(TOptions analyzeOptions)
             HashSet<string> targets = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
             foreach (string specifier in analyzeOptions.TargetFileSpecifiers)
             {
+                string normalizedSpecifier = specifier;
+
+                Uri uri;
+                if (Uri.TryCreate(specifier, UriKind.RelativeOrAbsolute, out uri))
+                {
+                    if (uri.IsFile || uri.IsUnc)
+                    {
+                        normalizedSpecifier = uri.LocalPath;
+                    }
```

Comment:
Sure, but we'll let the file specifier logic sort this out, otherwise we'll have two layers of error reporting